src/Form/AddressType.php

```diff
@@ -56,6 +56,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'data_class' => Address::class,
                 'error_bubbling' => false,
                 'disable_fields' => false,
+                'label' => false,
```

Comment:
This form is used in lots of places. Are you sure it's not breaking anything ? Why is it needed ? You can override this option for the specific form you want that. You can also just use `form_widget` in twig so that the label is not rendered.